### DISCUSSION TITLE
How to use csv exporter module?

### DISCUSSION THREAD

**Discussion by vichaya08** [SkypLabs/probequest#60] [Q&A]
I wonder how to use csv exporter in this tool. I try to use -o filename to export the csv file and I can't find any file in the system. any advice for me ? Thanks in advance!

**Comment by SkypLabs** [comment]
Hi @vichaya08,

At the moment, ProbeQuest only creates and populates the CSV file when exiting or when the size defined by your operating system's buffering policy has been reached.

Now that you mention it, it would be better to use a text stream instead of the current way of functioning. I will open a ticket to track this work. Thanks for raising this issue!
